calcom/cal.com
# chore: [app-router-migration 31] migrate the "/auth/setup" page

## What does this PR do?

- This PR migrates the `/auth/setup` page group to the app directory (which runs under the App Router).
- The migrated page exists under `/future/auth/setup` and is accessible at all times.

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change


## How should this be tested?

There should be no changes between the old setup and new setup routes in terms of content and no downgrading in terms of performance.

## Mandatory Tasks
